dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -631,6 +694,118 @@ std::unordered_map<TableID, SelectQueryInfo> DAGStorageInterpreter::generateSele
     return ret;
 }
 
+bool DAGStorageInterpreter::retryForBatchCopOrMPP(
```

Comment:
not a good name since it actually does not retry inside this function, maybe rename it to `checkRetriableForBatchCopOrMPP`?

Reply:
lgtm，renamed.